pkg/proxy/winkernel/proxier.go

```diff
@@ -436,11 +436,7 @@ func (proxier *Proxier) onServiceMapChange(svcPortName *proxy.ServicePortName) {
 // returns a new proxy.Endpoint which abstracts a endpointInfo
 func (proxier *Proxier) newEndpointInfo(baseInfo *proxy.BaseEndpointInfo, _ *proxy.ServicePortName) proxy.Endpoint {
 
-	portNumber, err := baseInfo.Port()
-
-	if err != nil {
-		portNumber = 0
-	}
+	portNumber := baseInfo.Port()
```

Comment:
you could just do this inline below, now that you don't need to deal with an error (like it does with `ip: baseInfo.IP()`)

Reply:
yes, I overlooked it.